requirements.yml

```diff
@@ -10,7 +10,7 @@
   version: 1.1.0
 
 - name: openmicroscopy.network
-  version: 1.1.1
+  version: 1.1.0
```

Comment:
Happy to push a 1.1.2 if possible.

Reply:
I don't know what made the `1.1.1` broken. If it's obvious, we can do that.

Reply:
While trying to test the `web-proxy` playbook, I realized this role version change currently breaks the playbook because `network_additional_param` is not defined. It is possible to overcome this by defining an empty `network_additional_param` var. Fixing the role for both usages would certainly be preferrable. I assume https://github.com/openmicroscopy/ansible-role-network/issues/4 is the placeholder for the discussion?